src/Glpi/Kernel/Kernel.php

```diff
@@ -88,7 +88,7 @@ public function getProjectDir(): string
 
     public function getCacheDir(): string
     {
-        return GLPI_CACHE_DIR . '/app/' . $this->environment;
+        return GLPI_CACHE_DIR . '/' . GLPI_FILES_VERSION . '/app';
```

Comment:
SHoudl not env kept? CacheManager class comments tells `inside GLPI_CACHE_DIR/GLPI_FILES_VERSION/$context.`

Reply:
With the latest commit, the cache dir will be different for each environment, e.g. `files/_cache/11.0.0-rc1-abcdef123-production`